### ISSUE TITLE
Annual project reviews

### ISSUE THREAD

**Issue by dthaler** [confidential-computing/governance#18]
At the San Francisco meeting, someone raised the issue that the Project Progression Policy document says "Projects will be reviewed on an annual basis" but little more information.   What topics should the TAC want to cover in such a review?

I have two suggestions for such a review:
1) Review whether any answers to the project's submission template have changed, and if so, review the new answers.  Thus, a rep from the project would be responsible for presenting any deltas to the template answers, since the last review.
2) Review the project's progression status.  E.g., is it already ready to move to another progression level?  Is it on track at the current level?  Is any action needed from the TAC (e.g., change in or addition to any project mentor(s))?

**Comment by dthaler** [comment]
Currently the Project Progression Policy document contains a placeholder section:

> IV. Annual Review Process
> The TAC shall develop an annual review process to determine whether projects are in the stage that accurately reflects their needs and goals.

**Comment by spjohnso** [comment]
We should merge this issue with #13 ?

**Comment by dthaler** [comment]
> We should merge this issue with #13 ?

Agree it is a dup, I missed that.  But I see you closed #13 so it's merged into this one I guess :)

**Comment by dthaler** [comment]
And a third suggestion:
3.  Review any budget allocations relevant to the project, and whether they need any adjustments.

**Comment by spjohnso** [comment]
Do we need to establish a calendar for project reviews... where would we publish this?

**Comment by dcmiddle** [comment]
One year is a pretty long time. Consider a little higher frequency to keep the projects and TAC connected. The updates are a chance to not only identify issues but also share learnings... What's working or not working, be it in the community or technically. 
Hyperledger uses quarterly updates which tend to be reviewed offline and only discussion bubbling up from comments needs to take place live in the TAC equivalent. A template and examples reports are here:
https://wiki.hyperledger.org/display/TSC/Project+Status+Updates

Looking at the CCC priorities I recommend a list to include:

- Significant changes to answers from the original [submission](https://github.com/confidential-computing/governance/blob/master/project-submission-template.md), esp.:
-- License
-- Scope
- Community health
-- Maintainer diversity
-- CoC issues or trends
- Project Progression status
-- Desire to graduate to next stage
- Issues for the TAC
- Accomplishments
-- Releases, Features, or Findings
- Plans
-- Releases, Features, or Findings

**Comment by dthaler** [comment]
This issue is addressed in PR #45.   One future point raised there is that not all answers are relevant to a running project. Once the template is complete, we should consider which ones we think should be reviewed.

**Comment by dcmiddle** [comment]
Addressed in #45
